src/corppa/poetry_detection/annotation/process_adjudication_data.py

```diff
@@ -0,0 +1,176 @@
+import argparse
```

Comment:
Please add a short docstring documenting briefly what this script does, what it is for, and one example of how to run it.

Here's how I ran it locally on the round 2 adjudication data:
```sh
python src/corppa/poetry_detection/annotation/process_adjudication_data.py allround2.jsonl round2_pages round2_excerpts
```

It wasn't obvious to me from the help information that I should have probably named the second param `round2_pages.jsonl` and the second one `round2_excerpts.csv`. I wasn't sure why we need both outputs or what the goals are for the different outputs.

Reply:
Just want to verify that the parser help descriptions are clear. For example, `output_pages` was described as "Filename where extracted pages data (JSONL file) should be written".

Reply:
"extracted pages data" was ambiguous to me in this context, I didn't know what it included or how it differed from the other output until poking around in the files a bit.

Maybe something like annotations aggregated/grouped by page  ?

Reply:
Does it make more sense in context now?

Reply:
I'm fine with changing the name later, but not something worth burning more of my R&D time on.

Reply:
thanks for updating the help text, it is clearer now